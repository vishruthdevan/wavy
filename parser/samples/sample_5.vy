aud1 = load("input1.wav")
format = "aac"

if (format == 'aac') {
    changeFormat(aud1, 'aac')
    export(aud1, 'output1.aac')
} else {
    changeFormat(aud1, 'mp3')
    export(aud1, 'output1.mp3')
}
