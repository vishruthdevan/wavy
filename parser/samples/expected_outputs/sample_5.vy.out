PROGRAM
ASSIGNMENT
├── IDENTIFIER (aud1)
├── =
└── CALL_EXPRESSION
    ├── FUNCTION
    │   IDENTIFIER (load)
    └── ARGUMENTS
        └── STRING
            └── input1.wav
ASSIGNMENT
├── IDENTIFIER (format)
├── =
└── STRING
      └── aac
IF_STATEMENT
├── CONDITION
INFIX
│     ├── IDENTIFIER (format)
│     ├── ==
│     └── STRING
│          └── aac
├── CONSEQUENCE
│   BLOCK
│   ├── CALL_EXPRESSION
│   │   ├── FUNCTION
│   │   │   IDENTIFIER (changeFormat)
│   │   └── ARGUMENTS
│   │       ├── IDENTIFIER (aud1)
│   │       └── STRING
│   │           └── aac
│   └── CALL_EXPRESSION
│       ├── FUNCTION
│       │   IDENTIFIER (export)
│       └── ARGUMENTS
│           ├── IDENTIFIER (aud1)
│           └── STRING
│               └── output1.aac
└── ALTERNATIVE
    BLOCK
      ├── CALL_EXPRESSION
      │   ├── FUNCTION
      │   │   IDENTIFIER (changeFormat)
      │   └── ARGUMENTS
      │       ├── IDENTIFIER (aud1)
      │       └── STRING
      │           └── mp3
      └── CALL_EXPRESSION
          ├── FUNCTION
          │   IDENTIFIER (export)
          └── ARGUMENTS
              ├── IDENTIFIER (aud1)
              └── STRING
                  └── output1.mp3
